src/www/app/tunnel.ts

```diff
@@ -42,4 +42,15 @@ export interface Tunnel {
 
   // Sets a listener, to be called when the tunnel status changes.
   onStatusChange(listener: (status: TunnelStatus) => void): void;
+
+  /**
+   * Get a value indicating whether this tunnel is able to install required services at runtime.
+   */
+  readonly canInstallServices: boolean;
```

Comment:
This method is not needed. The code that throws the configuration error that triggers the installation only runs on the platforms that support installation.
Let's remove this method and always trigger the install flow on `SystemConfigurationException`